service/alerter/service.go

```diff
@@ -1,19 +1,17 @@
 package alerter
 
 import (
-	"encoding/json"
 	"fmt"
 	"strings"
 	"time"
 
-	"github.com/giantswarm/awstpr"
+	"github.com/giantswarm/apiextensions/pkg/clientset/versioned"
```

Comment:
Mainly just deleting old crap in this PR. Only changes were for the alerter service to use `apiextensions`.